presto-native-execution/presto_cpp/main/http/HttpClient.h

```diff
@@ -215,7 +216,9 @@ class HttpClient : public std::enable_shared_from_this<HttpClient> {
 class RequestBuilder {
  public:
   RequestBuilder() {
-    headers_.setHTTPVersion(1, 1);
+    if (!SystemConfig::instance()->exchangeHttp2Enabled()) {
```

Comment:
https://github.com/facebook/proxygen/blob/691f58c1164610df1a3efb206f89d81fa065fb0d/proxygen/httpserver/ResponseBuilder.h#L70
This might not be necessary as Proxygen hardcode the version number. Version number isn't used.